lib/eventboss.rb

```diff
@@ -37,17 +37,16 @@ def publisher(event_name, opts = {})
     end
 
     def sender(event_name, destination_app, options = {})
-      queue_name = Queue.build_name(
+      queue = Queue.build(
         destination: destination_app,
         source: configuration.eventboss_app_name,
-        event: event_name,
-        env: env,
-        generic: options[:generic]
+        event_name: event_name,
+        env: env
```

Comment:
Why you're not taking `options[:generic]` into consideration?

Reply:
good point, will add.